docs/reference/api/docker_remote_api_v1.24.md

```diff
@@ -3003,6 +3022,15 @@ Return low-level information on the volume `name`
 -   **404** - no such volume
 -   **500** - server error
 
+**JSON parameters in response**:
```

Comment:
I added a short introduction, and made some changes, let me know if those look
good to you:

```Markdown
**JSON fields in response**:

The following fields can be returned in the API response. Empty fields, or
fields that are not supported by the volume's driver may be omitted in the
response.

- **Name** - Name of the volume.
- **Driver** - Name of the volume driver used by the volume.
- **Mountpoint** - Mount path of the volume on the host.
- **Status** - Low-level details about the volume, provided by the volume driver.
    Details are returned as a map with key/value pairs: `{"key":"value" [,"key2":"value2"]}`.
    The `Status` field is optional, and is omitted if the volume driver does not
    support this feature.
- **Labels** - Labels set on the volume, specified as a map: `{"key":"value" [,"key2":"value2"]}`
- **Scope** - Scope describes the level at which the volume exists, can be one of
    `global` for cluster-wide or `local` for machine level. The default is `local`.
```